daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -194,7 +194,7 @@ class DFDLTestSuite private[tdml] (
   // That avoids creating the test suites repeatedly, but also leaks memory unless
   // you have an @AfterClass shutdown method in the object that calls runner.reset() at end.
   //
-  // @deprecated("2016-12-30", "Use Runner(...) instead.")
+  // @deprecated("Use Runner(...) instead.", "2016-12-30")
```

Comment:
Does the deprecation annotation work inside a comment?

Reply:
No. We want to deprecate this, but we backed off as there were too many changes required. Some day we can put it back in though. There's a JIRA ticket for updating all tests to use Runner.